sematic/db/migrations/20221215212459_external_resources.sql

```diff
@@ -0,0 +1,26 @@
+-- migrate:up
+
+CREATE TABLE external_resources (
+    id character(32) NOT NULL,
+    resource_state TEXT NOT NULL,
+    locally_allocated BOOLEAN NOT NULL,
+    status_message TEXT NOT NULL,
+    last_updated_epoch_seconds int8 NOT NULL,
+    type_serialization JSONB NOT NULL,
+    value_serialization JSONB NOT NULL,
+    history_serializations JSONB NOT NULL,
+    PRIMARY KEY (id)
+);
+
+CREATE TABLE run_external_resources (
```

Comment:
In general, I have been using Rails conventions for the DB schema. In Rails, join tables are named as follows:
```
<plural-model1-name>_<plural-model2-name>
```
which would yield
```
runs_external_resources
```
here.

Not a blocker, but maybe we can maintain that.

https://guides.rubyonrails.org/active_record_migrations.html#creating-a-join-table

Reply:
done